test/library/packages/Python/correctness/basicTypes.chpl

```diff
@@ -30,6 +30,11 @@ proc roundTripFunction(func: borrowed) {
   testType(bool, true);
   testType(bytes, b"hello");
 
+  var tup1: (int, string, bool) = (17, "blah", false);
+  testType(tup1.type, tup1);
+  var tup2: 4*int = (5, 4, 3, 2);
+  testType(tup2.type, tup2);
+
```

Comment:
another edge case here would be something like `3*owned Value` or `(int, owned Value, real)`. Also, nested tuples and lists would be good to test here

Reply:
I guess there is a memLeaks test of lists inside tuples, so that ones less important here

Reply:
```chapel
var tup3: 3*owned Value? = (new Value(func.interpreter, 1),
                              new Value(func.interpreter, 2),
                              new Value(func.interpreter, 1));
testType(tup3.type, tup3);
```

```
basicTypes.chpl:14: error: halt reached - Unsupported toPython type: 'owned Value?'
```
I don't think this is something that works in other structures, either, tbh . . .

Reply:
update your branch, it should. I fixed that recently 🙂

Reply:
The new failure mode is an assertion failure on the `==` comparison with the returned value and the original.  I believe that's because classes only return true for `==` when the class pointer matches.  Okay if I skip adding this check?

Reply:
I think its fine to skip. Maybe worthwhile to add this test case standalone somewhere else, but its not that important